build.gradle

```diff
@@ -86,7 +86,8 @@ rat {
     'photon-ml/LINEAR_REGRESSION*/**',
     'photon-ml/LOGISTIC_REGRESSION*/**',
     'photon-ml/POISSON_REGRESSION*/**',
-    'photon-test/src/test/resources/**'
+    'photon-test/src/test/resources/**',
+    'tests.sh'
```

Comment:
I asked the Open Source department about this and they agree that it's not mandatory, since we include the license and notice files for the distribution. However, they're split on whether it is best practice to include a header anyway.

I'll add it, better safe than sorry